.cspell.json

```
@@ -720,6 +720,7 @@
     "fopen",
     "fopenmp",
     "forestgreen",
+    "fortelion",
```

If the original spelling is in uppercase, it's a good idea to add the same.
Adding it in uppercase to the dictionary will also work for lowercase `fortelion`.

```suggestion
    "FORTELION",
```

---

Thanks for the advice, I'll do that.